src/contracts/interfaces/IBLSRegistryCoordinatorWithIndices.sol

```diff
@@ -42,6 +42,8 @@ interface IBLSRegistryCoordinatorWithIndices is ISignatureUtils, IRegistryCoordi
 
     event ChurnApproverUpdated(address churnApprover);
 
+    event EjectorUpdated(address ejector);
```

Comment:
AFAIK it's more standard to emit both past address and the new one, but I think this is OK

Reply:
c5501a83